websites/#/라프텔/presence.ts

```diff
@@ -0,0 +1,171 @@
+const presence = new Presence({
+	clientId: "1133347072200949770",
+});
+
+let prevData = "",
+	AnimeData: Detail = {},
+	AnimeDataEpisode: Episode = {};
+
+type Detail = {
+	id?: string;
+	name?: string;
+	img?: string;
+	is_ending?: boolean;
+	animation_info?: {
+		air_year_quarter?: string;
+	};
+	meta_info?: {
+		avg_rating?: string;
+	};
+};
+
+type Episode = {
+	id?: string;
+	title?: string;
+	subject?: string;
+	episode_num?: string;
+};
+
+function getQuery() {
+	return JSON.parse(
+		`{"${decodeURI(location.search.substring(1))
+			.replaceAll('"', '\\"')
+			.replaceAll("&", '","')
+			.replaceAll("=", '":"')}"}`
+	);
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/PDh4ncE.jpg",
+	};
+
+	if (window.location.pathname === "/") {
```

Comment:
Properties of `location` are used many times. Destructure them for simplicity/consistency with other presences. In addition, we prefer `document.location` over `window.location`